src/main/java/frc/robot/commands/ColorSelectionCommand.java

```diff
@@ -0,0 +1,102 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+import edu.wpi.first.wpilibj.util.Color;
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorMatch;
+
+
+public class ColorSelectionCommand extends Command {
+  final Color kBlueTarget = ColorMatch.makeColor(0, 255, 255);
+  final Color kGreenTarget = ColorMatch.makeColor(0, 255, 0);
+  final Color kRedTarget = ColorMatch.makeColor(255, 0, 0);
+  final Color kYellowTarget = ColorMatch.makeColor(255, 255, 0);
+  private final I2C.Port i2cPort = I2C.Port.kOnboard;
+  String colorString;
+  double counter = 0;
+  final ColorSensorV3 m_colorSensor = new ColorSensorV3(i2cPort);
+     final ColorMatch m_colorMatcher = new ColorMatch();
```

Comment:
why is there an indent here?

Reply:
because errors in vscode

Reply:
this one makes no sense to me 
it's just being wack

Reply:
what did the error say?